kickoff/static/treestatus.js

```diff
@@ -0,0 +1,49 @@
+function _isTreeForThunderbird(treeName) {
+    return treeName.indexOf('comm') > -1;
+}
+
+function getTreeStatusUrl(branchName) {
+    var treeName = branchName.split('/')[1];
+    if (_isTreeForThunderbird(treeName)) {
+        treeName += '-thunderbird';
+    }
+
+    return 'https://treestatus.mozilla.org/' + treeName + '?format=json';
+}
+
+function fetchTreeStatus(branchElement, newerBranchName) {
+    var branchName = newerBranchName ? newerBranchName : branchElement.val();
+
+    var warningElement = branchElement.siblings('.help').find('.warning');
+    warningElement.text('');
+
+    $.ajax({
+        url: getTreeStatusUrl(branchName),
+    }).done(function(results) {
+        var status = results.status;
+        if (status !== 'open' && status !== 'approval required') {
```

Comment:
I don't believe so. Let's says `status === 'approval required'`. The condition would be: `true || false`. Hence it'd display the message anyway.